src/components/QueryEditor.tsx

```diff
@@ -153,22 +151,24 @@ export class QueryEditor extends PureComponent<Props, State> {
     let property = null;
     const listProperties = this.props.query.properties?.map((selectedProperty, index) => {
       property = (
-        <div className={'gf-form'}>
-          <InlineFormLabel width={8} tooltip={'Add select'}>
-            Select
-          </InlineFormLabel>
-          <Select
-            value={allProperties.find((item) => item.value?.name === this.props.query.properties?.[index].name)}
-            isClearable={true}
-            placeholder="(Property)"
-            onChange={(item) => this.onPropertyChange(item, index)}
-            onBlur={this.props.onRunQuery}
-            options={allProperties}
-            isSearchable={false}
-          />
-          <Button variant={'secondary'} onClick={() => this.removeProperty(index)}>
-            -
-          </Button>
+        <div className="gf-form-inline">
+          <div className={'gf-form'}>
+            <InlineFormLabel width={8} tooltip={'Add select'}>
+              Select
+            </InlineFormLabel>
+            <Select
+              value={allProperties.find((item) => item.value?.name === this.props.query.properties?.[index].name)}
+              isClearable={true}
```

Comment:
The `clearable true` introduces an little 'x' to clear the selected fields that does nothing when clicking on it. 
We should either handle the click on the clear button of set clearable to false

The same behaviour applies to the other select components